Sources/TestingInternals/include/Includes.h

```diff
@@ -27,6 +27,7 @@
 /// - Note: Avoid including headers that aren't actually used.
 
 #include <errno.h>
+#include <signal.h>
```

Comment:
Does this one not need a __has_include or similar guard? E.g. for non-POSIX embedded environments?

Reply:
No, `signal.h` is a C standard library header, so it should always be available.

Reply:
A coworker pointed out to me one embedded environment without the header (ignoring here the presence of signals as a feature either way.) So I added a check.